internal/jobcontainers/path.go

```diff
@@ -71,6 +71,13 @@ func getApplicationName(commandLine, workingDirectory, pathEnv string) (string,
 		searchPath string
 		result     string
 	)
+
+	// Asking for a relative path via "./", so trim the front. We handle relative paths such as "path/to/thing" and "path\\to\\thing", so just
+	// get rid of the prefix.
+	if len(commandLine) >= 3 && commandLine[0:2] == "./" || commandLine[0:2] == ".\\" {
```

Comment:
wouldn't `filepath.Clean` work here?

Reply:
More than likely, but then I don't get to add all of this fun logic :). Joking of course, this slipped my mind